File: client/src/app/_models/uploadComicDto.ts
import { ApprovalStatusComic } from '../_extensions/enumHelper';
import { GenreForUploadComicDto } from './genreForUploadComicDto';

export interface UploadComicDto {
  id: number | null;
  authorName: string;
  name: string;
  isFeatured: boolean;
  desc: string;
  status: boolean;
  isCompleted: boolean;
  mainImage: string;
  rate: number;
  nOReviews: number;
  totalChapter: number;
  newestChapter: string;
  approvalStatus: ApprovalStatusComic;
  selectedGenres: GenreForUploadComicDto[];
}

File: API/Dtos/UploadComicDto.cs
using API.Helpers;

namespace API.Dtos
{
    public class UploadComicDto
    {
        public int Id { get; set; }
        public string AuthorName { get; set; }
        public string Name { get; set; }
        public bool IsFeatured { get; set; }
        public string Desc { get; set; }
        public bool Status { get; set; }
        public bool IsCompleted { get; set; }
        public string MainImage { get; set; }
        public int Rate { get; set; }
        public int NOReviews { get; set; }
        public int TotalChapter { get; set; }
        public string NewestChapter { get; set; }
        public ApprovalStatusComic ApprovalStatus { get; set; }
        public List<GenreForUploadComicDto> SelectedGenres { get; set; }
    }
}